tests/test_release_transpiler_argument_types.py

```diff
@@ -0,0 +1,565 @@
+import argparse
+import sys
+from contextlib import contextmanager
+from os.path import abspath, dirname
+from typing import List
+
+import pytest
+
+from komodo.release_transpiler import main as release_transpiler_main
+
+RELEASE_TRANSPILER_FILE_PATH = abspath(
+    dirname(dirname(abspath(__file__))) + "/komodo/release_transpiler.py"
+)
+VALID_RELEASE_FOLDER = abspath(dirname(__file__) + "/data/test_releases")
+VALID_RELEASE_BASE = "2020.01.a1"
+VALID_OVERRIDE_MAPPING_FILE = abspath(
+    dirname(dirname(__file__)) + "/examples/stable.yml"
+)
+VALID_PYTHON_COORDS = "3.6,3.8"
+
+ARGUMENT_PARSING_ERROR = SystemExit
```

Comment:
I think you can skip having these as constants, it should be just as readable in the test code to just use the exceptions as-is.